src/main/java/io/thinkit/edc/client/connector/Constants.java

```diff
@@ -2,12 +2,18 @@
 
 public interface Constants {
     String EDC_NAMESPACE = "https://w3id.org/edc/v0.0.1/ns/";
-
+    String DCAT_NAMESPACE = "https://www.w3.org/ns/dcat/";
+    String DCT_NAMESPACE = "https://purl.org/dc/terms/";
     String ODRL_NAMESPACE = "http://www.w3.org/ns/odrl/2/";
+    String DSCPACE_NAMESPACE = "https://w3id.org/dspace/v0.8/";
 
     String ID = "@id";
     String TYPE = "@type";
     String VALUE = "@value";
     String CONTEXT = "@context";
     String VOCAB = "@vocab";
+    String DCAT = "dcat";
+    String DCT = "dct";
+    String ODRL = "odrl";
+    String DSCPACE = "dspace";
```

Comment:
in the Json-LD specs these are called "prefix", so we could call the constants `XXX_PREFIX`